tooling/minimize/src/ty.rs

```diff
@@ -78,6 +78,10 @@ pub fn translate_ty<'tcx>(ty: rs::Ty<'tcx>, tcx: rs::TyCtxt<'tcx>) -> Type {
             let elem = GcCow::new(translate_ty(*ty, tcx));
             Type::Array { elem, count }
         }
+        rs::TyKind::FnPtr(_) => {
+            // For now we use the C ABI for everything since that's what `spawn` needs...
+            Type::Ptr(PtrType::FnPtr(CallingConvention::C))
```

Comment:
We already have logic to translate Rust calling conventions to MiniRust ones, please use that here.

Reply:
I am unsure how I even get the Conv. The FnPtr type only provides `PolyFnSig` and there we only find an ABI. The only *very* ugly way I see to convert between them would be through `&str`.
Other than Rust and C almost none would match up.

Reply:
This would look something like this...
`translate_calling_convention(FromStr::from_str(sig.abi().name()).unwrap())`

Reply:
No no don't use a string please. :) You need to use `fn_abi_of_fn_ptr`. This should be similar to the existing code that uses `fn_abi_of_instance`, so maybe looking at that is helpful.